lib/ansible/modules/files/find.py

```diff
@@ -146,6 +146,7 @@
 - name: Find /var/log files equal or greater than 10 megabytes ending with .old or .log.gz
   find:
     paths: /var/log
+    use_regex: yes
```

Comment:
This isn't correct.  The `patters` in this task are not regex. They are shell/fnmatch patterns.